builds/jazz-build-module/sls-app/serverless-build-rules.yml

```diff
@@ -70,13 +70,13 @@ provider:
     sbr-constraint:
       sbr-enum:
         - nodejs8.10
+        - nodejs10.x
```

Comment:
is `nodejs10.x` a valid option - I mean the `.x`?

Reply:
Yep - https://docs.aws.amazon.com/en_pv/lambda/latest/dg/programming-model.html